bok_choy/browser.py

```diff
@@ -204,8 +205,13 @@ def browser_check_func():
             LOGGER.debug('Failed to instantiate browser: ' + err.strerror)
             return False, None
 
+        except WebDriverException as err:
+            msg = str(err)
```

Comment:
Looks like err.strerror will bomb out if there is not a valid string raised in the exception. Fixing this to use the same pattern I was using.